### ISSUE TITLE
BDB does not show full company information when navigating from the BDB list

### ISSUE THREAD

**Issue by kotlarz** [webkom/lego-webapp#847]
Example:
- Visit https://abakus.no/bdb
![1](https://user-images.githubusercontent.com/14221489/32058899-d6553860-ba6a-11e7-902b-abb6784affce.png)
- Click on a Company
- Information is missing
![2](https://user-images.githubusercontent.com/14221489/32058913-dc086016-ba6a-11e7-9a4b-aa3d75ea15d5.png)
- Refresh the company page to get the detailed view

**Comment by finnss** [comment]
It displays the information it has until the detail version is fetched. I'm unable to reproduce this bug

**Comment by odinuge** [comment]
This is a caching-bug. I think i have found the issue, will create a PR asap :smile: 

https://github.com/webkom/lego-webapp/blob/52e57b903138ed45c314aa320a34057cc23863a9/app/routes/bdb/BdbDetailRoute.js#L26

Is there by there a reason that you fetch the company _after_ the semesters, and not? No data from the first request is used in the second one.. The chaining is btw. the reason the companyDetail isn't fetched, since the `.then` isn't executed (not your, or bdb's fault tho)..

```jsx
const loadData = ({ params: { companyId } }, dispatch) =>
  Promise.all([
    dispatch(fetchSemesters()),
    dispatch(fetchEventsForCompany(companyId)),
    dispatch(fetchAdmin(companyId))
  ]);

```

**Comment by finnss** [comment]
I chained them becuase when I do `selectCompany` after `fetchAdmin` succeeds, I map semesterStatuses from having `semester: <some companySemester's id>` to `semester: { <the full companySemester object>`. Thus, I figured I needed the full `companySemester` objects to be available. I haven't actually tested how this acts when the actions aren't chained, but I didn't see a disadvantage to chaining so I figured I might as well circumvent the issue altogether.

Looking forward to seeing the issue fixed in #872 <3
